src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -272,6 +272,14 @@ public static bool CanRemoveParentheses(this ParenthesizedExpressionSyntax node,
                 return true;
             }
 
+            // Range expression
+            // x[(y)..] => x[y..]
+            if (node.Parent is RangeExpressionSyntax)
+            {
+                if (expression.IsKind(SyntaxKind.ElementAccessExpression))
```

Comment:
this doesn't seem right.  i think the appropriate question would be is the expr in teh parens higher precedence than the rangeepression?  if so, parens can be removed.  if not, they have to say.